lib/travis/build/script/r.rb

```diff
@@ -0,0 +1,384 @@
+module Travis
+  module Build
+    class Script
+      class R < Script
+        DEFAULTS = {
+          # Basic config options
+          cran: 'http://cran.rstudio.com',
+          warnings_are_errors: false,
+          # Dependencies (installed in this order)
+          apt_packages: [],
+          brew_packages: [],
+          r_binary_packages: [],
+          r_packages: [],
+          bioc_packages: [],
+          r_github_packages: [],
+          # Build/test options
+          r_build_args: '',
+          r_check_args: '--as-cran',
+          r_check_cran_incoming: false,
+          r_check_revdep: false,
+          # Heavy dependencies
+          pandoc: true,
+          pandoc_version: '1.13.1',
+          # Bioconductor
+          bioc: 'http://bioconductor.org/biocLite.R',
+          bioc_required: false,
+          bioc_use_devel: false,
+        }
+
+        def initialize(data)
+          # TODO(craigcitro): Is there a way to avoid explicitly
+          # naming arguments here?
+          super
+          @devtools_installed = false
+          @bioc_installed = false
+        end
+
+        def export
+          super
+          sh.export 'TRAVIS_R_VERSION', 'release', echo: false
+        end
+
+        def setup
+          super
+
+          # TODO(craigcitro): Confirm that these do, in fact, print as
+          # green. (They're yellow under vagrant.)
+          sh.echo 'R for Travis-CI is not officially supported, ' +
+                  'but is community maintained.', ansi: :green
+          sh.echo 'Please file any issues using the following link',
+                  ansi: :green
+          sh.echo '  https://github.com/travis-ci/travis-ci/issues' +
+                  '/new?labels=community:r', ansi: :green
+          sh.echo 'and mention \`@craigcitro\`, \`@eddelbuettel\` and ' +
```

Comment:
Maybe this should be: "And include `cc @craigcitro, @eddelbuettel, @hadley` in the issue. That way people can copy and paste more easily